util/rna-seq-reprocessing-tags.json

```diff
@@ -0,0 +1,19 @@
+[
+  {
+    "ResourceType": "capacity-reservation",
+    "Tags": [
```

Comment:
Good idea -- I'll give that a try.

Reply:
@zaro0508 I added the ability to pull the tags from the instance that and then realized that's not going to work. Since the way that I get to make capacity reservations is for my personal account to assume the AWSCapacityReservationRole, I have to move all my personal information to one of the toil instances to run the script. We log into these as ubuntu (the jumpbox) or root (the leader), so I really don't want to put my personal keys out there. The way I currently have it written, I can just assume the role and run the script from my laptop. Unless you have a suggestion for a good workaround for this problem, I'm going to let it stand as written.

Reply:
@zaro0508 The solution we agreed on to provide two options for supplying tags was added in f931245, i.e., 1. user supplies tags json or 2. user provides an instance id, and the script pulls the tags for that instance and applies a subset of those the capacity reservation.